Sources/SwiftSyntax/SyntaxData.swift

```diff
@@ -49,7 +49,7 @@ struct AbsoluteSyntaxInfo {
 }
 
 /// Provides a stable and unique identity for `Syntax` nodes.
-struct NodeIdentity: Hashable {
+public struct NodeIdentity: Hashable {
```

Comment:
Apologies for the back-and-forth on this -- can you rename this to `SyntaxIdentifier` so it looks more like something from the `Syntax` namespace? We don't often use `node` in API.

Reply:
SGTM, done!